eos/form-factors/baryonic.cc

```diff
@@ -26,11 +27,16 @@ namespace eos
 {
     /* J=1/2^+ -> J=1/2^+ Processes */
 
-    /* Lambda_b -> Lambda */
+    /* Lambda_b -> Lambda, Lambda_b -> Lambda_c */
```

Comment:
Please split the instantiation of ```LambdaBToLambdaC``` from ```LambdaBToLambda```. Also, you need to instantiate
**all** the members to avoid the error seen in the ```clang``` CI failure.